src/components/interactive-map.js

```diff
@@ -90,14 +90,14 @@ export default class InteractiveMap extends PureComponent {
           ref: map => {
             this._map = map;
           },
-          children: mapVisible && this.props.children
+          children: null
         })),
         createElement('div', {
-          key: 'map-children',
-          style: {position: 'absolute', left: 0, top: 0}
-        },
-          !mapVisible && this.props.children
-        )
+          key: 'map-overlays',
+          className: 'overlays',
```

Comment:
It doesn't have to. The class name of the container makes it easier to do css styling. Having the same class name makes css reusable for both static and interactive maps. If we decide it's not necessary then both should be removed?